examples/admipex9CB_example.jl

```diff
@@ -0,0 +1,80 @@
+# This example is translated to Julia from the example of the CPLEX Callable library, admipex9.c.
+
+using CPLEX
+
+env = CPLEX.Env()
+# Build the model and read the problem from the .mps file.
+path = joinpath(dirname(@__FILE__), "sentoy.mps")
+model = CPLEX.Model(env, path)
+CPLEX.read_model(model, path)
+# Add this to all integer models. Need to change cpx_solve.jl L4.
+model.has_int = true
```

Comment:
This can be removed once https://github.com/JuliaOpt/CPLEX.jl/pull/211 is merged